draft-ietf-quic-transport.md

```diff
@@ -1641,6 +1641,14 @@ of connection establishment.  By giving the client a different connection ID to
 use, a server can cause the connection to be routed to a server instance with
 more resources available for new connections.
 
+If a server receives a client Initial with a unverifiable Retry token,
+it knows the client will not accept another Retry token.  It can either
+proceed with the handshake without verifying the token or immediately close
+({{immediate-close}}) the connection with an connection error of
+INVALID_TOKEN to cause the handshake to fail quickly instead of waiting
+for the client to timeout.  When a client receives a CONNECTION_CLOSE
+with an error of INVALID_TOKEN, it MAY create a new connection.
```

Comment:
This isn't a very useful "MAY", because we couldn't stop a client from doing this.  We never say anything about making other connections anywhere else.

Reply:
SG, sentence removed.